gravitee-node-notifier/src/main/java/io/gravitee/node/notifier/trigger/NotificationTrigger.java

```diff
@@ -87,16 +91,22 @@ public NotificationTrigger(
     }
 
     public void start() {
+        started.set(true);
         scheduleNextAttempt();
     }
 
     private void scheduleNextAttempt() {
-        this.scheduledTaskId = this.vertx.setTimer(computeNextAttempt(), this);
+        if (started.get()) {
+            this.scheduledTaskId = this.vertx.setTimer(computeNextAttempt(), this);
+        }
     }
 
     public void stop() {
-        if (this.scheduledTaskId != null && this.vertx.cancelTimer(this.scheduledTaskId)) {
+        if (started.get() && this.scheduledTaskId != null) {
```

Comment:
Hum... I have the feeling it doesn't solve the issue. I was thinking of using  `compareAndSet()` to ensure no concurrent actions.